frb_example/flutter_via_create/rust/src/frb_generated.io.rs

```diff
@@ -68,7 +68,7 @@ pub extern "C" fn cst_new_list_prim_u_8(len: i32) -> *mut wire_cst_list_prim_u_8
 }
 
 #[repr(C)]
-#[derive(Clone)]
+#[derive(Clone, Copy)]
 pub struct wire_cst_list_prim_u_8 {
     ptr: *mut u8,
     len: i32,
```

Comment:
nit: I am a bit worried - should we make this Copy? Since it contains raw pointers, and copying it may unwantedly make more references than we realize.

Reply:
This was a consequence of requiring Clone on structs, which means that MaybeUninit<T> requires T to be Copy. If we aren't using Clone for anything we could remove both.

Reply:
I see. Then maybe we can avoid MaybeUninit below?

Reply:
I checked back and it's not so simple, it seems that the Clone derive requires every union field to implement Copy which is stronger than a union's usual requirement of either ManuallyDrop or Copy.

I also tried removing both derives but we do have some functions that require Clone, so we're at an impasse. I think it's actually fine to implement Copy for both, since until they have been deserialized they are still packs of pointers and numbers which are trivially copyable without loss of safety; the real unsafety of pointers is at the point of dereferencing.

Reply:
So, if I understand correctly, this means that, Copy is needed even without the usage of MaybeUninit?

If so, maybe we have to use Copy.

Reply:
Here's the [playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=a064bde04abb0b443100b00651d7dc78) for the issue, but yes the only way for a union to implement Clone is to also implement Copy.